Main/src/Main/CoffeeMachineSimulatorMain.java

```diff
@@ -0,0 +1,26 @@
+package Main;
+import java.util.Scanner;
+
+public class CoffeeMachineSimulatorMain {
+
+
+    public static String getCurrentLine;
+
+    //printing actions, call myMethod
+    public static void main(String[] args) {
+        System.out.println("Write action (buy, fill, take, remaining, exit):");
+        getCurrentLine = getString();
+        CoffeeMachineSimulator coffeeMachineSimulator = new CoffeeMachineSimulator();
+        coffeeMachineSimulator.myMethod(getCurrentLine);
+
+
+    }
+
+    //method getString read input from console
+    public static String getString() {
```

Comment:
@GayaneGulanyan jan please change method name

Reply:
Why, name is not meaningful, or there is other reason?

Reply:
@nsimonyan 
@Edvard777